src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -44,11 +45,48 @@ public void Initialize(IEventSource eventSource)
             eventSource.ErrorRaised += new BuildErrorEventHandler(eventSource_ErrorRaised);
             // Cancelled
             Console.CancelKeyPress += new ConsoleCancelEventHandler(console_CancelKeyPressed);
-            // Initialize FancyLoggerBuffer
-            FancyLoggerBuffer.Initialize();
-            // TODO: Fix. First line does not appear at top. Leaving empty line for now
-            FancyLoggerBuffer.WriteNewLine(string.Empty);
-            FancyLoggerBuffer.Render();
+            
+            Task.Run(() =>
+            {
```

Comment:
Can you pull these contents into a named method please? It's a bit long IMO to be in a lambda.